frontend/src/components/JiraDashboard/components/WidgetJira.js

```diff
@@ -0,0 +1,198 @@
+                                                    borderRadius: "4px",
+                                                }),
+                                            },
+                                        }}
+                                        error={
+                                            errMessage.length > 0 ? true : false
+                                        }
+                                    />
+                                </FormControl>
+                            </div>
+                        </>
+                    ) : (
+                        <>
+                            <div className="jiraWid__filter">
+                                <FormControl>
+                                    <Select
+                                        size={SIZE.compact}
+                                        options={filters}
+                                        value={filterValue}
+                                        placeholder="Select Filter"
+                                        onChange={handleFilter}
+                                        overrides={{
+                                            ControlContainer: {
+                                                style: ({ $theme }) => ({
+                                                    borderRadius: "4px",
+                                                }),
+                                            },
+                                        }}
+                                    />
+                                </FormControl>
+                            </div>
+
+                            <div className="jiraWid__filter">
+                                <FormControl>
+                                    <Select
+                                        size={SIZE.compact}
+                                        options={employeeDetails}
+                                        value={employeeFilterValue}
+                                        placeholder="Select Assignee"
+                                        onChange={handleEmployeeFilter}
+                                        overrides={{
+                                            ControlContainer: {
+                                                style: ({ $theme }) => ({
+                                                    borderRadius: "4px",
+                                                }),
+                                            },
+                                        }}
+                                    />
+                                </FormControl>
+                            </div>
+                        </>
+                    )}
+                    {basicMode === false ? (
+                        <Button
+                            onClick={handleClick}
+                            className="btn--blue"
+                            size={SIZE.compact}
+                            overrides={{
```

Comment:
same as above. create separate components for basic filter mode and advanced filter mode